modules/es-extensions/publisher/extensions/app/greg_publisher/themes/default/js/validate.js

```diff
@@ -17,21 +17,35 @@
  *
  */
 
-function validate() {
+var validate = function () {
 
     var password = $('#value').val();
     var passwordVerify = $('#valueVerify').val();
 
-    if(password != passwordVerify){
-        alert("Re-entered password does not match");
+    if (password != passwordVerify) {
+        messages.alertError("Re-entered password does not match");
         return false;
     }
     else {
+        var ajaxURL = caramel.context + '/pages/password?type=server';
+        $.ajax({
+            url: ajaxURL,
+            type: 'GET',
+            async: false,
+            success: function (data) {
+                messages.alertSuccess("Property Saved Successfully");
+
+            },
+            error: function (data) {
+                messages.alertSuccess("Property Could Not Be Saved Successfully");
+
```

Comment:
Hi Imesh,
Thanks For pointing this out. I have fixed this and all the other places as well in the same PR.

Thanks!